### PR TITLE
feature: format image size

### PR DESCRIPTION
**Pull request by Ace-Tang**
Signed-off-by: Ace-Tang <aceapril@126.com>

**1.Describe what this PR did**

format image size

**2.Does this pull request fix one issue?** 

**3.Describe how you did it**

before image size format 
```
$ sudo ./pouch images 
IMAGE ID             IMAGE NAME                                               SIZE
bbc3a0323522         docker.io/library/busybox:latest                         2699
```

with this patch

```
$ sudo ./pouch images
IMAGE ID             IMAGE NAME                                               SIZE
bbc3a0323522         docker.io/library/busybox:latest                         2.64 KB
```

**4.Describe how to verify it**

**5.Special notes for reviews**

### CODE REVIEW

File: ctrd/image.go [hunk 1]
```diff
@@ -250,7 +250,19 @@ outer:
 }
 
 func formatSize(size int64) string {
-	return strconv.FormatInt(size, 10)
+	// we consider image size less than 1024 GB
```
Comment (allencloud) [inline ctrd/image.go]: We can add unit test for this function definitely.

**Comment by Ace-Tang** [comment]
have updated, @allencloud

File: pkg/utils/utils.go [hunk 2]
```diff
@@ -1,9 +1,28 @@
...
 func If(cond bool, v1, v2 interface{}) interface{} {
 	if cond {
 		return v1
 	}
 	return v2
 }
+
+// FormatSize format image size to B/KB/MB/GB
+func FormatSize(size int64) string {
+	// we consider image size less than 1024 GB
```
Comment (allencloud) [inline pkg/utils/utils.go]: In code, what about numbers larger than 1024MB?
Maybe could be a little bit robust.

File: pkg/utils/utils_test.go [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestFormatSize(t *testing.T) {
+	assert := assert.New(t)
+	kvs := map[int64]string{
+		1024:          "1.00 KB",
+		1024000:       "1000.00 KB",
+		1024000000000: "953.67 GB",
```
Comment (allencloud) [inline pkg/utils/utils_test.go]: I am wondering if we could add more test cases: negative ones, 0, and positive ones.

Comment (Ace-Tang) [reply pkg/utils/utils_test.go]: yes, has completed cases

**Comment by skoo87** [comment]
LGTM
